doc/source/parametric-methods-to-compare-two-means.md

```diff
@@ -0,0 +1,260 @@
+
+This is what makes it possible to estimate the population mean in a random variable whose distribution is normal through the Student's t distribution, being the parameter extracted from a small sample and its population variance is also unknown.
+
+On the other hand, when there is equality between the variances (homoscedasticity) of two groups that are contrasted, it is when a hypothesis test between their two means is used and its random variable has a normal distribution.
+
+## Comparison between two independent means
+
+In general, it is usually contrasted if there is a significant difference between the mean of two different and independent populations and for this, the usual thing is for two independent samples to use the Student's t test.
+
+This test is used to compare the means of a certain quantitative and continuous variable obtained for two categories determined by a qualitative variable and is based on the calculation of a statistic, which takes into account the difference between the means to be compared, in addition to their standard error and for this, the following formula is used:
+
+![Formula](_static/images/parametric-methods-to-compare-two-means/formula-1.jpg)
+
+Where <img src="https://latex.codecogs.com/png.image?\dpi{110}\overline{X1}" title="https://latex.codecogs.com/png.image?\dpi{110}\overline{X1}" /> and (S1)<sup>2 </sup> are the mean and variance of the first sample and in the same way, <img src="https://latex.codecogs.com/png.image?\dpi{110}\overline{X2}" title="https://latex.codecogs.com/png.image?\dpi{110}\overline{X2}" /> and (S2)<sup>2 </sup> are the mean and variance of the second sample.
+The assumption of the null hypothesis is that both means are equal, so the value of t will also be equal to zero (0). It is important to emphasize that the further away t is from this value, the less likely it is that both equal means are due to chance.
+
+The conditions that are required to be able to apply this test are the following:
+
+1.- Both groups must be independent and this implies that each individual must belong to only one of the two groups and that in addition, it has no relationship with the individuals of the other group.
+
+2.- The variable to be considered must be continuous and have a normal distribution in both groups.
+
+3.- The homoscedasticity assumption must be fulfilled, that is, the variance must be the same in the two groups.
+To verify if the distribution of the variable is normal in both groups, the Shapiro Wilk test can be performed for when the samples are small (less than 50). For large samples, it is best to use the Kolmogorov Smirnof test and in case of having a large sample and not knowing both the standard deviation and the population mean, it is advisable to use this same test but with the modification of Lilliefors.
```

Comment:
```suggestion

To verify if the distribution of the variable is normal in both groups, the Shapiro Wilk test can be performed for when the samples are small (less than 50). For large samples, it is best to use the Kolmogorov Smirnof test and in case of having a large sample and not knowing both the standard deviation and the population mean, it is advisable to use this same test but with the modification of Lilliefors.
```